docs/try-it-out.mdx

```diff
@@ -15,16 +15,16 @@ doc_type: getting-started
 displayed_sidebar: mainSidebar
 ---
 
-Explore what's possible using the Beyond Identity platform before having to write any code.
+See what you can do with Beyond Identity before you write any code.
 
-The three steps below allow you to: 
+The steps that follow allow you to, as a developer: 
```

Comment:
Since this is "Developer Documentation," I think it helps to focus the content on the developer.